### PR TITLE
Localise profile overlay numeric statistics

### PR DESCRIPTION
**Pull request by Game4all**
Follow-up to #13921.

This localises formatting of most numerical statistics on the user profile overlay.

comparision

### CODE REVIEW

**Review by frenzibyte — CHANGES_REQUESTED** [review]
The usage for `UsersStrings.ShowExtraTopRanksPpWeight` should've been here but it seems that the current strings classes force parameters to be `string`s.

Will update them, and potentially update the localistaion analyzer tool while at it.

File: osu.Game/Screens/Play/Break/BreakInfoLine.cs [hunk 1]
```diff
@@ -63,7 +64,7 @@ private void currentValueChanged(ValueChangedEvent<T> e)
             valueText.Text = newText;
         }
 
-        protected virtual string Format(T count)
+        protected virtual LocalisableString Format(T count)
```
[the remark this answers is not in the document]
Comment (Game4all) [reply osu.Game/Screens/Play/Break/BreakInfoLine.cs:67]: This should be treated in its own localisation pass of gameplay elements I guess.

[the remark this answers is not in the document]
Comment (frenzibyte) [reply osu.Game/Screens/Play/Break/BreakInfoLine.cs:67]: Sounds fine.

**Review by frenzibyte — APPROVED** [review]

**Review by frenzibyte — APPROVED** [review]
The `UsersStrings.ShowExtraTopRanksPpWeight` case can be tackled on separately once the [prerequisite](https://github.com/ppy/osu-resources/pull/139) for it gets merged.

File: osu.Game/Overlays/Profile/Header/Components/RankGraph.cs [hunk 2]
```diff
@@ -65,7 +66,7 @@ protected override object GetTooltipContent(int index, int rank)
 
             return new TooltipDisplayContent
             {
-                Rank = $"#{rank:N0}",
+                Rank = rank.ToLocalisableString("\\##,##0"),
```
Comment (smoogipoo) [inline osu.Game/Overlays/Profile/Header/Components/RankGraph.cs:69]: Intentional change?

Comment (frenzibyte) [reply osu.Game/Overlays/Profile/Header/Components/RankGraph.cs:69]: It doesn't look to be localisable in its previous form, unless there's some way to prepend a `#` while still using the `N0` format.

Comment (smoogipoo) [reply osu.Game/Overlays/Profile/Header/Components/RankGraph.cs:69]: https://github.com/ppy/osu-web/blob/2574ce3139557b6cc5acd468c7a7ad4abc40670b/resources/assets/coffee/react/profile-page/rank.coffee#L16-L19 Hmm I suppose web does the same thing.

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**